src/main/java/net/snowflake/ingest/streaming/internal/BlobBuilder.java

```diff
@@ -90,6 +91,8 @@ static <T> Blob constructBlobAndMetadata(
         final byte[] compressedChunkData;
         final int chunkLength;
         final int compressedChunkDataSize;
+        long extendedMetadataSize = -1;
+        long metadataSize = -1;
```

Comment:
Add Constants.EP_EXTENDED_METDATA_SIZE_UNKNOWN and Constants.EP_METADATA_SIZE_UNKNOWN, just like we have it for NDV_UNKNOWN and NV_UNKNOWN

Reply:
Added.